lib/dealership.rb

```diff
@@ -0,0 +1,37 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if inventory_count > 0
+      true
+    else
+      false
+    end
+  end
+
+  def cars_by_make(make)
+    same_make = []
+    @inventory.each do |inventory|
+      if inventory.car.make == make
+      @same_make << @inventory.car
+
+     end
+     return same_make
```

Comment:
This `return` is placed inside of your `each` block - it should be moved down to after line 33. Right now it returns far too early, or not at all.